phoenix-core/src/it/java/org/apache/phoenix/end2end/CreateTableIT.java

```diff
@@ -708,4 +705,52 @@ public void testCreateTableWithUpdateCacheFrequencyAttrib() throws Exception {
         }
       }
     }
+
+    @Test
```

Comment:
I also added the test for createTable with Namespace Mapping enabled. This test fails against the original logic.